mobile.css

```diff
@@ -33,7 +33,6 @@ body {
   align-items: center;
```

Comment:
- [ ] You have done a good job with the mobile menu :100: but there is a style issue because when I scroll down I can actually see the content below and the mobile menu it's supposed to be on top of the content. I think you could use the `position: fixed` property to maintain the mobile menu on top even when scrolling :wink: :pushpin: 
![image](https://user-images.githubusercontent.com/81522435/146803705-41c0ce7c-4e3b-4171-88f5-ce44a3a8bc06.png)

- [ ] Also, according to the Figma design the mobile menu should have some transparency. I would suggest adressing this style issue in order to match the template :smiley: :raised_hands: 

| Figma design | Your design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/81522435/146803568-ac6db959-ece6-4a2f-818b-016eea789918.png) | ![image](https://user-images.githubusercontent.com/81522435/146803590-a0c9eb95-a052-4ea4-8534-241cc6bad8b3.png) |